pkg/controller/installation/products/threescale/reconciler.go

```diff
@@ -258,45 +264,12 @@ func (r *Reconciler) reconcileSMTPCredentials(ctx context.Context, inst *v1alpha
 }
 
 func (r *Reconciler) reconcileComponents(ctx context.Context, serverClient pkgclient.Client) (v1alpha1.StatusPhase, error) {
-	bucket := &v1.Secret{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: s3BucketSecretName,
-		},
-	}
-
-	err := serverClient.Get(ctx, pkgclient.ObjectKey{Name: bucket.Name, Namespace: r.ConfigManager.GetOperatorNamespace()}, bucket)
+	fss, err := r.getBlobStorageFileStorageSpec(ctx, serverClient)
```

Comment:
do any of the README docs need to change to account for these changes. for example, we currently tell a user to pre-seed a secret, is that required anymore, and if they do pre-seed it, is it used?

Reply:
Ok will update :+1: